CONTRIBUTING.md

```diff
@@ -299,6 +300,55 @@ CRS:
 
 </details>
 
+### Test via building Euro Data Cube Public Collection website locally
+
+Test the script before a merge request is helpful for the reviewing process, making a contribution be publish sooner. Below is a step-by-step guide to build the site locally.
+
+<details> 
+   <summary>Linux & Mac</summary>
+   <li>Create a .env file with the following content in the repository.
+
+   ```
+   GIT_HUB_COLLECTIONS_REPO=eurodatacube/public-collections
+   GIT_HUB_COLLECTIONS_BRANCH=main
+   COLLECTIONS_BROWSER_ROOT_URL="https://collections.eurodatacube.com/"
+   ```
+
+   </li>
+   <li>Open Terminal.</li>
+   <li>Run <code>export $(xargs <.env)</code>.</li>
```

Comment:
I suggest to move the `export $(xargs <.env)` to jsut before the `npm run build` command, since you need it everytime you reboot the terminal

Reply:
Split the section into `Prerequisites` and `Building your site locally`